python/fastexcel/_fastexcel.pyi

```diff
@@ -70,6 +80,12 @@ class _ExcelSheet:
         """The visibility of the sheet"""
     def to_arrow(self) -> pa.RecordBatch:
         """Converts the sheet to a pyarrow `RecordBatch`"""
+    def to_arrow_with_errors(self) -> tuple[pa.RecordBatch, CellErrors]:
```

Comment:
suggestion: how about returning `tuple[pa.RecordBatch, CellErrors | None]` ?

This would allow to write code such as

```py

record_batch, errors = sheet.to_arrow_with_errors()

if errors:
    # Custom error handling here
```

Reply:
Updated